blog/secadvisor.md

```diff
@@ -0,0 +1,290 @@
+    <img src="/assets/images/blog/capture-future-1.png"/>
+  </a>
+</p>
+
+
+Make sure you are on the "Captures" view and press on "Create".
+
+Fill in the "Targets" fields by placing the cursor in the first "Interface" field and then use the mouse to point to your network endpoint that you want to capture.
+To capture a particular flow end-to-end, fill in the second "Interface" field in a simmilar manner with the other network endpoint.
+Then press the "Start" button on the GUI.
+
+Alternatively, you can start the capture of flows from the command line with a commond like the following:
+
+```
+skydive client capture create --gremlin "G.V().Has('Type', 'device', 'Name', 'eth0')" --type pcap 
+```
+
+The command specifies to capture all flows that match the gremlin expression - in this case, all devices that have name 'eth0'.
+
+### Generate Flows
+
+Generate some network traffic on the interfaces you specified to capture.
+For example, run some iperf traffic between entities and capture the flow via the Skydive GUI.
+
+Run iperf server:
+
+```
+iperf3 -s
+```
+
+Either on the same machine or on another machine, run the iperf client:
+
+```
+iperf3 -c <address-of-iperfserever> -t 1000
+```
+
+It is also possible to generate network traffic from the Skydive GUI.
+Go the "Generator" page and specify the source and destination nodes between which to generate the network traffic, as well as the characterization of the flow you want to generate.
+
+### Observe Security Advisor objects being created
+
+Objects are created in the Object Store only if there is flow information to be saved.
+
+The output log of secadvisor should show that an object is being sent to the object store about once per minute.
+Check your bucket in your object store to verify that you see a new object about once per minute.
+Stop the secadvisor or stop captures in the GUI to stop the creation of objects in the object store.
+
+
+
+## Multiple pipelines
+
+It is possible to run multiple pipelines simultaneously. Prepare a separate <n>.yml for each set of subnets or whatever configuration you want. Then run a separate instance of the Security Advisor (or other pipeline) using each of the <n>.yml configuration files. In this way it is possible to perform different filtering for different purposes or tenants.
+
+## Trouble-shooting common problems
+
+If the secadvisor log shows a "connection refused" error, verify that the proper address of the Skydive analyzer is specified under "analyzers".
+
+If the secadvisor log shows a credentials error, verify that the credential fields (<"access_key", "secret_key"> for minio, and <"api_key", "iam_endpoint"> for IBM COS) are properly set in the secadvisor.yml file.
+
+If the secadvisor log shows an overflow and states that flows were discarded, check that max_flow_array_size is defined to some reasonable positive number - at least the number of flows you expect to capture.
+
+Verify that host_id is "".
+
```

Comment:
please remove empty lines at the end of the file

Reply:
OK